index.html

```diff
@@ -5,21 +5,21 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <script defer src="menu.js"></script>
     <title>Willy's portfolio</title>
 </head>
 <body>
     <div class="display">
-        <header>
-            <div class="flexend wide">
-                <nav><a class="menubar" href="#" onclick="showMenu()"><img class="menu-button" src="images/menu.png" alt="menu"></a>
-                    <ul class="hormenu">
+        <header class="flexend wide" id="header">           
+                <nav class='navbar' id="nav">
+                    <button type="button" class="menubar" id="menubar"><img class="open" id='open' src="./images/menu.png" alt="menu-open"><img class="close" id='close' src="./images/close.png" alt="menu-open"></button>
+                    <ul class="hormenu" id="hormenu">
                         <li class="link"><a href="#Hello">Hello</a></li>
                         <li class="link"><a href="#Projects">Projects</a></li>
                         <li class="link"><a href="#About">About</a></li>
                         <li class="link"><a href="#Contact">Contact</a></li>
                     </ul>
-                </nav>
-            </div>
+                </nav>            
         </header>
     </div>  
```

Comment:
- [ ] Kindly adjust the size of the modal to take 100% of the screen's width and height to match the Figma design.
- [ ] Please make sure that you can't scroll down to the page's content when the modal is opened. (You can use `position: fixed` so that )
- [ ] Please make sure the modal is closed and hidden from view after the user clicks on any of the nav links.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/162819569-7bb7f345-ae21-4dd4-8f01-89b359d56b09.png) | ![image](https://user-images.githubusercontent.com/62290545/162819454-116469a0-2cd2-46e5-887d-43f83be985d2.png) |